### PR TITLE
Added freefare_version() function

### PR DESCRIPTION
**Pull request by broth-itk**
This is to address

**Add a function to get lib version #74**

### CODE REVIEW

File: libfreefare/freefare.c [hunk 1]
```diff
@@ -212,6 +212,20 @@ freefare_free_tag(FreefareTag tag)
     }
 }
 
+/*
+ * Return lib version
+ */
+const char *
+freefare_version(void)
+{
+#ifdef GIT_REVISION
```
Comment (smortex) [inline libfreefare/freefare.c]: I don't think we define `GIT_VERSION` anywhere.

File: libfreefare/freefare.3 [hunk 2]
```diff
@@ -117,6 +120,10 @@ can be gathered using the
 and
 .Fn freefare_get_tag_friendly_name
 functions.
+.Pp
+Use
+.Fn freefare_version
+to obtain a library version string
```
Comment (smortex) [inline libfreefare/freefare.3]: :100: for thinking about the man page!

For consistency, can you reword this to something like `The freefare_version function returns the version of library.`?

**Comment by smortex** [comment]
I just pushed a few commits to `master` that should fix the Travis-CI issue you see with CMake.

You can get them in your branch by rebasing.  Assuming you have 2 remotes, `origin` is the "official" repo and `broth-itk` is your fork:

1. `git fetch --all` (get all commits locally)
2. `git rebase origin/master` (add the locally missing commits from `origin/master` before your local work.  Your local commits will be modified, so you will have to use `-f` to push them next time).

**Comment by broth-itk** [comment]
Is there anything to do from my side in order to get this pull request merged?
